Code/Bench/fingerprint.cpp

```diff
@@ -0,0 +1,23 @@
+#include <catch2/catch_all.hpp>
+#include <string>
+
+#include "bench_common.hpp"
+
+#include <GraphMol/SmilesParse/SmilesParse.h>
+#include <GraphMol/Fingerprints/MorganFingerprints.h>
+
+using namespace RDKit;
+
+TEST_CASE("MorganFingerprints::getFingerprint", "[fingerprint]") {
```

Comment:
Please use the fingerprint generator for this one; that's the recommended way to create fingerprints now